packages/trader/src/Assets/SvgComponents/trade_explanations/img-turbos.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 272 148" fill="none" xmlns="http://www.w3.org/2000/svg"><path width="272" height="148" d="M268 0H4C1.79 0 0 1.779 0 3.973v139.054C0 145.221 1.79 147 4 147h264c2.209 0 4-1.779 4-3.973V3.973C272 1.779 270.209 0 268 0Z" fill="var(--general-section-1)"/><path width="272" height="148" d="M157.12 89.6h32.77c3.22 0 5.83 2.405 5.83 5.365 0 2.97-2.61 5.363-5.83 5.363H40.6c-3.21 0-5.82-2.394-5.82-5.364 0-2.96 2.6-5.363 5.83-5.363h64.08c3.22 0 5.83-2.404 5.83-5.364s-2.6-5.363-5.83-5.363H55.16v-.05c-2.91-.337-5.1-2.622-5.1-5.323 0-2.702 2.19-4.987 5.1-5.324v-.04h20.4c3.21 0 5.82-2.404 5.82-5.364 0-2.97-2.6-5.373-5.83-5.373H48.61c-3.22 0-5.83-2.404-5.83-5.364 0-2.97 2.6-5.363 5.83-5.363h138.37c3.22 0 5.83 2.394 5.83 5.363 0 2.96-2.61 5.364-5.83 5.364h-53.9c-3.21 0-5.82 2.404-5.82 5.364s2.61 5.363 5.83 5.363h32.04c3.22 0 5.83 2.414 5.83 5.374 0 2.96-2.61 5.363-5.83 5.363h-17.48c-3.21 0-5.82 2.404-5.82 5.364 0 2.97 2.6 5.373 5.82 5.373h9.47Zm-126-42.937c3.23 0 5.84 2.404 5.84 5.364 0 2.98-2.61 5.373-5.83 5.373s-5.83-2.404-5.83-5.364c0-2.97 2.61-5.363 5.83-5.363l-.01-.01Zm136.92-21.454h40.79c3.21 0 5.82 2.394 5.82 5.364 0 2.97-2.6 5.363-5.82 5.363h-40.79c-3.21 0-5.82-2.404-5.82-5.363 0-2.97 2.6-5.364 5.82-5.364Zm-24.76 0c3.22 0 5.83 2.394 5.83 5.364 0 2.97-2.61 5.363-5.83 5.363s-5.82-2.404-5.82-5.363c0-2.97 2.6-5.364 5.82-5.364Zm70.64 21.454h10.2c3.22 0 5.83 2.404 5.83 5.364 0 2.98-2.61 5.373-5.83 5.373h-10.2c-3.21 0-5.82-2.404-5.82-5.364 0-2.97 2.6-5.363 5.82-5.363v-.01Zm-25.49 21.474h21.85c3.22 0 5.83 2.404 5.83 5.364s-2.6 5.363-5.83 5.363h-21.84c-3.22 0-5.83-2.404-5.83-5.364s2.6-5.363 5.83-5.363h-.01Zm20.4 21.464h32.04c3.22 0 5.83 2.404 5.83 5.364 0 2.97-2.61 5.363-5.83 5.363h-32.04c-3.22 0-5.83-2.394-5.83-5.364 0-2.96 2.61-5.363 5.83-5.363Zm-139.1 21.464h65.54c3.22 0 5.83 2.404 5.83 5.363 0 2.97-2.61 5.364-5.83 5.364H69.73c-3.22 0-5.83-2.394-5.83-5.364 0-2.969 2.6-5.363 5.83-5.363Zm87.39 0h10.2c3.21 0 5.82 2.404 5.82 5.363 0 2.97-2.6 5.364-5.82 5.364h-10.2c-3.22 0-5.83-2.394-5.83-5.364 0-2.969 2.61-5.363 5.83-5.363Z" fill="var(--fill-normal)"/><path d="M208 85.25h-7a1 1 0 0 1-1-1v-8.697l-28 26.25-16-15-12 11.25V87.447l12-11.25 16 15 22.343-20.947H185a1 1 0 0 1-1-1v-6.5h24v22.5Z" fill="#FF444F"/><path d="M144 45.875h8v7.5h-8v-7.5Zm16 0h8v7.5h-8v-7.5Zm16 0h8v7.5h-8v-7.5ZM192 44h16v11.25h-16l-4-5.625L192 44ZM64 94.283h8v7.45h-8v-7.45Zm16 0h8v7.45h-8v-7.45Zm16 0h8v7.45h-8v-7.45Zm16-1.863h16v11.175h-16l-4-5.587 4-5.587Z" fill="#85ACB0"/><path d="M128 62.624h-7a1 1 0 0 0-1 1v8.63L92 46.183 76 61.081 64 49.907v10.535l12 11.174 16-14.899 22.343 20.805H105a1 1 0 0 0-1 1v6.45h24V62.624Z" fill="#FF444F"/></svg>
```

Comment:
```suggestion
<svg viewBox="0 0 272 148" fill="none" xmlns="http://www.w3.org/2000/svg"><path width="272" height="148" d="M268 0H4C1.79 0 0 1.779 0 3.973v139.054C0 145.221 1.79 147 4 147h264c2.209 0 4-1.779 4-3.973V3.973C272 1.779 270.209 0 268 0Z" fill="var(--fill-color3)"/><path width="272" height="148" d="M157.12 89.6h32.77c3.22 0 5.83 2.405 5.83 5.365 0 2.97-2.61 5.363-5.83 5.363H40.6c-3.21 0-5.82-2.394-5.82-5.364 0-2.96 2.6-5.363 5.83-5.363h64.08c3.22 0 5.83-2.404 5.83-5.364s-2.6-5.363-5.83-5.363H55.16v-.05c-2.91-.337-5.1-2.622-5.1-5.323 0-2.702 2.19-4.987 5.1-5.324v-.04h20.4c3.21 0 5.82-2.404 5.82-5.364 0-2.97-2.6-5.373-5.83-5.373H48.61c-3.22 0-5.83-2.404-5.83-5.364 0-2.97 2.6-5.363 5.83-5.363h138.37c3.22 0 5.83 2.394 5.83 5.363 0 2.96-2.61 5.364-5.83 5.364h-53.9c-3.21 0-5.82 2.404-5.82 5.364s2.61 5.363 5.83 5.363h32.04c3.22 0 5.83 2.414 5.83 5.374 0 2.96-2.61 5.363-5.83 5.363h-17.48c-3.21 0-5.82 2.404-5.82 5.364 0 2.97 2.6 5.373 5.82 5.373h9.47Zm-126-42.937c3.23 0 5.84 2.404 5.84 5.364 0 2.98-2.61 5.373-5.83 5.373s-5.83-2.404-5.83-5.364c0-2.97 2.61-5.363 5.83-5.363l-.01-.01Zm136.92-21.454h40.79c3.21 0 5.82 2.394 5.82 5.364 0 2.97-2.6 5.363-5.82 5.363h-40.79c-3.21 0-5.82-2.404-5.82-5.363 0-2.97 2.6-5.364 5.82-5.364Zm-24.76 0c3.22 0 5.83 2.394 5.83 5.364 0 2.97-2.61 5.363-5.83 5.363s-5.82-2.404-5.82-5.363c0-2.97 2.6-5.364 5.82-5.364Zm70.64 21.454h10.2c3.22 0 5.83 2.404 5.83 5.364 0 2.98-2.61 5.373-5.83 5.373h-10.2c-3.21 0-5.82-2.404-5.82-5.364 0-2.97 2.6-5.363 5.82-5.363v-.01Zm-25.49 21.474h21.85c3.22 0 5.83 2.404 5.83 5.364s-2.6 5.363-5.83 5.363h-21.84c-3.22 0-5.83-2.404-5.83-5.364s2.6-5.363 5.83-5.363h-.01Zm20.4 21.464h32.04c3.22 0 5.83 2.404 5.83 5.364 0 2.97-2.61 5.363-5.83 5.363h-32.04c-3.22 0-5.83-2.394-5.83-5.364 0-2.96 2.61-5.363 5.83-5.363Zm-139.1 21.464h65.54c3.22 0 5.83 2.404 5.83 5.363 0 2.97-2.61 5.364-5.83 5.364H69.73c-3.22 0-5.83-2.394-5.83-5.364 0-2.969 2.6-5.363 5.83-5.363Zm87.39 0h10.2c3.21 0 5.82 2.404 5.82 5.363 0 2.97-2.6 5.364-5.82 5.364h-10.2c-3.22 0-5.83-2.394-5.83-5.364 0-2.969 2.61-5.363 5.83-5.363Z" fill="var(--fill-normal)"/><path d="M208 85.25h-7a1 1 0 0 1-1-1v-8.697l-28 26.25-16-15-12 11.25V87.447l12-11.25 16 15 22.343-20.947H185a1 1 0 0 1-1-1v-6.5h24v22.5Z" fill="#FF444F"/><path d="M144 45.875h8v7.5h-8v-7.5Zm16 0h8v7.5h-8v-7.5Zm16 0h8v7.5h-8v-7.5ZM192 44h16v11.25h-16l-4-5.625L192 44ZM64 94.283h8v7.45h-8v-7.45Zm16 0h8v7.45h-8v-7.45Zm16 0h8v7.45h-8v-7.45Zm16-1.863h16v11.175h-16l-4-5.587 4-5.587Z" fill="#85ACB0"/><path d="M128 62.624h-7a1 1 0 0 0-1 1v8.63L92 46.183 76 61.081 64 49.907v10.535l12 11.174 16-14.899 22.343 20.805H105a1 1 0 0 0-1 1v6.45h24V62.624Z" fill="#FF444F"/></svg>
```

Reply:
Thank you for reviewing, but what you suggested change the color of image from light gray to white (what goes against the proposed design)

Reply:
We did it like it it had been done in other (already existing) pictures for contracts